impl/restart_integration_test.go

```diff
@@ -118,6 +132,11 @@ func TestRestartPush(t *testing.T) {
 				if event.Code == datatransfer.DataSent {
 					if channelState.Sent() > 0 {
 						sent <- channelState.Sent()
+						if rh.dt1.IsChannelMonitored(chid) {
```

Comment:
is there something we could do here to collect state and assert on it later?

Track `monitoredCount` `unmonitoredCount` and assert they're above some base number?